scripts/format_source.py

```diff
@@ -0,0 +1,11 @@
+import json
+from typing import List, cast
+
+
+def format_source(source: List[str]) -> List[str]:
+    """Due to Argo constraints if using the basemaps cli the source has a string that contains a list that needs to be split.
+    example: ["[\"s3://test/image_one.tiff, \"s3://test/image_two.tiff\"]"]
+    """
+    if len(source) == 1 and source[0].startswith("["):
+        return cast(List[str], json.loads(source[0]))
```

Comment:
I would `json.loads` anyways return the string if `loads` raises an exception. In your case, if `source[0]` is a `str` like "[my_string" it would fails. Or maybe we want it to fails, but it would be good to nicely catch the exception in that case.